### DISCUSSION TITLE
How to create react project?

### DISCUSSION THREAD

**Discussion by lawcompanion1** [community/community#39350] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to create a react project (cmd command)??

**Comment by ws-jm** [comment, marked as answer]
npx create-react-app my-react-app
This is command

**Reply by lawcompanion1** [reply]
Thanks ws-jm

**Reply by ws-jm** [reply]
Welcome

**Comment by SilasRodrigues19** [comment]
@lawcompanion1 Install NodeJS LTS version and after use the following command:

`npx create-react-app my-app`

**Comment by SarathAdhi** [comment]
@lawcompanion1 
Firstly install `NODE` on your machine https://nodejs.org/en/download/

Open `CMD` in the preferred folder and paste `npx create-react-app <your-app-name>`

**Comment by clever-web** [comment]
how to use Redux in React? give me an answer
